lib/atom.ts

```diff
@@ -0,0 +1,6 @@
+import type { Theme } from "react95/dist/types";
+
+import { atom } from "jotai";
+import originalTheme from "react95/dist/themes/original";
```

Comment:
Let's call this `original` instead, so that it's the same as the name exported from `react95/dist/themes`